src/languageservice/utils/strings.ts

```diff
@@ -87,3 +87,18 @@ export function getFirstNonWhitespaceCharacterAfterOffset(str: string, offset: n
   }
   return offset;
 }
+
+export function addIndentationToMultilineString(text: string, firstIndent: string, nextIndent: string): string {
```

Comment:
Unified function to indent multiline strings, this logic was written multiple times before